web/themes/joinup/prototype/sass/components/_details.sass

```diff
@@ -71,6 +71,10 @@
     color: $color-grey-medium
     font-weight: $font-weight-medium
 
+.details__element--single
```

Comment:
Since we change details "component":
- the avatar's head is cut:
![screenshot from 2017-09-27 17-05-27](https://user-images.githubusercontent.com/1923476/30921282-634fde18-a3a6-11e7-9aaa-30b5453bc20f.png)
- inline of adjustment elements:
![screenshot from 2017-09-27 17-06-44](https://user-images.githubusercontent.com/1923476/30921300-73b1758c-a3a6-11e7-98aa-76bd23768a69.png)

Reply:
"the avatar's head is cut:"
It is cut because of image style. The image there is rounded so image style have square shape.

"inline of adjustment elements:"
Hmm, I am not sure what should be changed there. I am cutting the location text in css, in the same way as for tiles.